test/ClangImporter/Inputs/enum-anon.h

```diff
@@ -1,17 +1,26 @@
-enum {
+#if defined(_WIN32)
+typedef __INTPTR_TYPE__ intptr_t;
+
+#define MAKE_UNSIGNED : unsigned
+#define MAKE_SIGNEDLL : intptr_t
+#else
+#define MAKE_UNSIGNED
+#endif
```

Comment:
These are testing different things. I'd rather change expectations than lose the tests for an untyped anonymous enum.

Reply:
Ugh, okay, I guess we need to duplicate the tests then.  Because with the `-verify` we can't match the behavioural differences.  Windows does signed enumerations by default where as Darwin/Linux do `unsigned` as the base type.

Reply:
Nope, not in the standard :-(